layout-management-api/src/main/java/org/exoplatform/layoutmanagement/service/LayoutManagementService.java

```diff
@@ -0,0 +1,6 @@
+package org.exoplatform.layoutmanagement.service;
+
+public interface LayoutManagementService {
```

Comment:
We need to check if this can be declared and implemented directly into NavigationService layer of gatein-portal